konfera/register/views.py

```diff
@@ -4,8 +4,37 @@
 
 from konfera.models.event import Event
 from konfera.models.ticket import REQUESTED
-from konfera.models.ticket_type import TicketType, VOLUNTEER, PUBLIC
-from konfera.register.forms import VolunteerRegistrationForm
+from konfera.models.ticket_type import TicketType, VOLUNTEER, PUBLIC, PRIVATE, ACTIVE
+from konfera.register.forms import RegistrationForm
+
+
+def private_registration(request, event_slug, ticket_uuid):
+    context = dict()
+    event = get_object_or_404(Event, slug=event_slug)
+    ticket_type = get_object_or_404(TicketType, event=event.id, uuid=ticket_uuid, accessibility=PRIVATE)
+    if ticket_type.status != ACTIVE:
+        messages.error(request, "This ticket type is not available")
+        return redirect('event_details', event_slug)
+
+    form = RegistrationForm(request.POST or None)
```

Comment:
If related ticket_type is in (VOLUNTEER, PRESS, AID) make description field required, for other ticket types it can be optional.